src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -11,6 +11,63 @@
   .text--truncate();
 }
 
+// Family portal specific styling
+.MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal {
+  border-radius: @size_m;
+  background-color: @neutral_white;
+  box-shadow: 0px @size_3xs 6px rgba(71, 76, 94, 0.3);
```

Comment:
Just curious! Could we use one of our [box shadow mixines](https://github.com/Clever/components/blob/master/src/less/box-shadow.less) here?

```
.boxShadow--medium();
```